tests/test_table_reset.py

```diff
@@ -0,0 +1,19 @@
+from sfbase import SFBaseTest
+from tap_tester.base_suite_tests.table_reset_test import TableResetTest
+
+
+class SFTableResetTest(TableResetTest, SFBaseTest):
+    """tap-salesforce Table reset test implementation
+    Currently tests only the stream with Incremental replication method"""
+
+    #reset_stream = 'User'
```

Comment:
Don't need this since we have it below.  Although, if you make this a property then this would work and you could remove the other one. I think that looks better but either way works.

Reply:
Made it as a property